thirdparty/lpeg/CMakeLists.txt

```diff
@@ -9,9 +9,10 @@ enable_language(C)
 assert_var_defined(CC)
 assert_var_defined(DYNLIB_CFLAGS)
 assert_var_defined(LUA_DIR)
-assert_var_defined(LUAJIT_LIB)
 
-set(BUILD_CMD_STR "${CC} ${DYNLIB_CFLAGS} -I${LUA_DIR}/src ${LUAJIT_LIB} -o lpeg.so lpcap.c lpcode.c lpprint.c lptree.c lpvm.c")
+# assuming lpeg.so is installed under $(OUTPUT_DIR)/rocks/lib/lua/5.1 and
+# libluajit.so is in $(OUTPUT_DIR)/libs
+set(BUILD_CMD_STR "${CC} ${DYNLIB_CFLAGS} -I${LUA_DIR}/src -Wl,-rpath,'../../../../libs' -lluajit -o lpeg.so lpcap.c lpcode.c lpprint.c lptree.c lpvm.c")
```

Comment:
I'm not quite following everything, but it would appear some some bionic versions [do seem to handle DT_RUNPATH entries](/android/platform_bionic/blob/master/linker/linker.cpp#L1542)... :?